### PR TITLE
quic: Quiche roll 20220225121042

### PR DESCRIPTION
**Pull request by danzh2010**
Commit Message: Update QUICHE from dd22dd1a7 to 50f15e7a5 https://github.com/google/quiche/compare/dd22dd1a7..50f15e7a5

Risk Level: low
Testing: existing tests passes
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: n/a

### CODE REVIEW

File: source/common/quic/platform/quiche_mem_slice_impl.h [hunk 1]
```diff
@@ -14,41 +14,41 @@
 
 #include "quiche/quic/core/quic_buffer_allocator.h"
 
-namespace quic {
+namespace quiche {
 
 // Implements the interface required by
 // https://quiche.googlesource.com/quiche/+/refs/heads/master/quic/platform/api/quic_mem_slice.h
```
Comment (RenjieTang) [inline source/common/quic/platform/quiche_mem_slice_impl.h:20]: We probably need to update this comment as well.

**Review by RenjieTang — APPROVED** [review]
LGTM mod one nit

File: source/common/quic/platform/quiche_mem_slice_impl.h (hunk 1 above)
Comment (danzh2010) [reply source/common/quic/platform/quiche_mem_slice_impl.h]: good point!

**Comment by danzh2010** [comment]
@htuch for dependency review.

**Comment by moderation** [comment]
Was travelling OS

/lgtm deps

**Review by alyssawilk — APPROVED** [review]

**Pull request merged**